src/build/supportedExtensions.js

```diff
@@ -96,9 +96,9 @@ exports.extensions = {
     { icon: 'image', extensions: ['jpeg', 'jpg', 'gif', 'png', 'bmp', 'ico'] },
     { icon: 'ionic', extensions: ['ionic.project'], filename: true },
     { icon: 'ionic', extensions: ['ionic.config.json'], filename: true },
-    { icon: 'jade', extensions: ['jade', 'pug', 'tag'] },
```

Comment:
Extensions that collide should be studied. Some can be solved using languageIds. @JimiC you removed `prolog` language in another PR, which is not right, as languageIds can be added via plugins. In the case of `Prolog` you can find the extension [here](https://marketplace.visualstudio.com/items?itemName=rebornix.prolog) and if you see [their contribution point ](https://github.com/rebornix/vscode-prolog/blob/master/package.json) you'll see that the id of prolog was correct.